### PR TITLE
JML- formatted the arguments for handler

### PR DESCRIPTION
**Pull request by murad-ali-MoJ**
Formated the secret value received from lambda so that the get_secret_value function can produce the correct output.

### CODE REVIEW

File: containers/jml-extract-lambda/CHANGELOG.md
Comment (jacobwoffenden) [inline containers/jml-extract-lambda/CHANGELOG.md:1]: You've overwritten `1.0.1`

**Review by jacobwoffenden — CHANGES_REQUESTED** [review]

**Review by murad-ali-MoJ — COMMENTED** [review]
The PR ready to merge

File: containers/jml-extract-lambda/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.2] - 2024-01-29
+
+### Changed
+
+- Change the handler to updated the secret it needs to change the image as well
```
Comment (jhpyke) [inline containers/jml-extract-lambda/CHANGELOG.md]: Hi,

Can you update this to be a clearer description of what this update adds? I'd be sure from reading this

File: containers/jml-extract-lambda/src/var/task/handler.py [hunk 2]
```diff
@@ -17,12 +17,23 @@ def handler(event, context):
...
     secrets_client = boto3.client("secretsmanager")
 
-    response = secrets_client.get_secret_value(SecretId=SECRET_ID)
+    secret_id = SECRET_ID.split("|")
+    secret_arn = secret_id[0]
+    secret_version = secret_id[1]
+
+    response = secrets_client.get_secret_value(
+        SecretId=secret_arn, VersionStage=secret_version
+    )
```
Comment (jhpyke) [inline containers/jml-extract-lambda/src/var/task/handler.py:26]: This looks like what I'd hope to see

**Review by jhpyke — CHANGES_REQUESTED** [review]
Code looks good - needs a clearer patch note on what changes we're making

File: containers/jml-extract-lambda/CHANGELOG.md (hunk 1 above)
Comment (murad-ali-MoJ) [reply containers/jml-extract-lambda/CHANGELOG.md]: upadted

File: containers/jml-extract-lambda/CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.2] - 2024-01-29
+
+### Changed
+
+- Trimmed secrets env var, converted to ARN and version for get_secret_value compatibility.
```
Comment (jhpyke) [inline containers/jml-extract-lambda/CHANGELOG.md]:
```suggestion
- Bugfix: resolved issue with incorrect parsing of ID strings
```

**Review by jhpyke — DISMISSED** [review]
Looks ready to ship - tested locally and verified email output

**Review by jacobwoffenden — APPROVED** [review]

**Pull request merged**